### PR TITLE
This seems to fix the issue with some JPG files not loading

### PR DESCRIPTION
**Pull request by devedse**
See issue https://github.com/oozcitak/exiflibrary/issues/60

I'm not sure if this is the right fix, but for me it seems to work.
For some reason it seems to go into this code twice:

image

See the issue for more information.

### CODE REVIEW

File: ExifLibrary/JPEGFile.cs [hunk 1]
```diff
@@ -495,6 +495,9 @@ private void ReadExifAPP1()
             int thumboffset = -1;
             int thumblength = 0;
             int thumbtype = -1;
+
+            bool hasFound1stIFDpointer = false;
```
Comment (oozcitak) [inline ExifLibrary/JPEGFile.cs]:
```suggestion
```

File: ExifLibrary/JPEGFile.cs [hunk 2]
```diff
@@ -596,11 +601,14 @@ private void ReadExifAPP1()
 
                 // 1st IFD pointer
                 int firstifdoffset = ifdoffset + 2 + 12 * fieldcount;
-                if (firstifdoffset + 4 <= header.Length)
+                if (firstifdoffset + 4 <= header.Length && hasFound1stIFDpointer == false)
```
Comment (oozcitak) [inline ExifLibrary/JPEGFile.cs]:
```suggestion
                if (firstifdoffset + 4 <= header.Length && currentifd == IFD.Zeroth)
```

File: ExifLibrary/ExifBitConverter.cs [hunk 3]
```diff
@@ -352,6 +370,21 @@ public static byte[] GetBytes(MathEx.Fraction32[] value, ByteOrder tobyteorder)
             }
             return data;
         }
+
+        /// <summary>
+        /// Converts the given array of 16-bit unsigned integers to an array of bytes.
```
Comment (oozcitak) [inline ExifLibrary/ExifBitConverter.cs]:
```suggestion
        /// Converts the given array of 16-bit signed integers to an array of bytes.
```

**Comment by devedse** [comment]
I've fixed the feedback you mentioned :). Feel free to merge it and release a new NuGet package. (I'll be waiting for it to fix some issues in my tool I've been developing)

**Comment by devedse** [comment]
@oozcitak , I should have fixed all the changes. Could you already merge it :)?

**Pull request merged**